Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/IdentitiesRepository.cs

```diff
@@ -55,6 +46,21 @@ public async Task<bool> Exists(IdentityAddress address, CancellationToken cancel
             .AnyAsync(i => i.Address == address, cancellationToken);
     }
 
+    public async Task<IEnumerable<Identity>> FindAllWithAddresses(CancellationToken cancellationToken, IEnumerable<IdentityAddress>? addresses, IdentityStatus? requestStatus, bool track = false)
+    {
+        var query = (track ? _identities : _readonlyIdentities)
+            .IncludeAll(_dbContext);
+
+        if (addresses.Any())
```

Comment:
Honestly...? The `addresses` property is obviously nullable. And the compiler even warns you about the potential usage of a null reference. And still you forget to check for `null`?